### PR TITLE
FormsCommunityToolkit: Added Animations Sample

### PR DESCRIPTION
**Pull request by jsuarezruiz**
Added Animations Sample that allows to test all the possibilities of the animations library.

### CODE REVIEW

File: Samples/Samples/Samples/Views/AnimationsPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+using Xamarin.Forms;
+
+namespace FormsCommunityToolkit.Samples.Views
+{
+    public partial class AnimationsPage : TabbedPage
+    {
+        public AnimationsPage()
+        {
+            InitializeComponent();
+            var test = new BeginAnimationBehavior();
```
Comment (Depechie) [inline Samples/Samples/Samples/Views/AnimationsPage.xaml.cs]: Why is this test variable being declared here?

File: src/Animations/Animations/Triggers/BeginAnimation.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+namespace FormsCommunityToolkit.Animations
+{
+    public class BeginAnimation : TriggerAction<VisualElement>
+    {
+        public AnimationBase Animation { get; set; }
+
+        protected override async void Invoke(VisualElement sender)
+        {
+            if (Animation != null)
```
Comment (Depechie) [inline src/Animations/Animations/Triggers/BeginAnimation.cs]: I know, it's hard ;) But please only positive checks. if( == null ) return;

File: Samples/Samples/Samples/Views/AnimationsPage.xaml.cs (hunk 1 above)
Comment (jsuarezruiz) [inline Samples/Samples/Samples/Views/AnimationsPage.xaml.cs]: Ooops. It was a test that has totally escaped!

File: src/Animations/Animations/Triggers/BeginAnimation.cs (hunk 2 above)
Comment (jsuarezruiz) [inline src/Animations/Animations/Triggers/BeginAnimation.cs]: Yes, no problem ;)

**Pull request merged**